src/components/MapFacetContents.js

```diff
@@ -210,7 +228,9 @@ class MapFacetContents extends React.Component<MapFacetContentsDefaultProps, Map
             key={`${value.longitude || 0},${value.latitude || 0}`}
             style={{ cursor: 'pointer' }}
           >
-            <Glyphicon glyph="map-marker" style={{ fontSize: '18px', color: '#2a689c' }} />
+            <OverlayTrigger overlay={<Tooltip id="tooltip-bottom">{bucket.count} {this.props.tooltip} in this region</Tooltip>}>
```

Comment:
It would be more readable to put the <Tooltip> tag on a new line, like this:
```
<OverlayTrigger overlay={(
  <Tooltip id="tooltip-bottom">
    {formattedTooltip}
  </Tooltip>
}}> 
  {locationPointer}
</OverlayTrigger>
```

Reply:
Done